docs/synapse/devguides/devops_cortex.rst

```diff
@@ -89,6 +89,10 @@ the Axon.
 0.1.x to 2.x.x Migration
 ------------------------
 
+.. warning::
+    The 0.1.x to 2.x.x migration tools will be removed in Synapse v2.9.0. The tools
+    will continue to be available in previous version of the 2.x.x releases.
```

Comment:
Maybe a quick line about the upgrade path being 0.1.x -> 2.8.X -> fully updated?

Reply:
@invisig0th I refined this a bit

Reply:
Test